src/Common/src/Interop/Unix/libcurl/Interop.libcurl_types.cs

```diff
@@ -125,6 +131,19 @@ internal static partial class CURLMSG
             internal const int CURLMSG_DONE = 1;
         }
 
+        // AUTH related constants
+        internal static partial class CURLAUTH
+        {
+            internal const ulong None = 0;
+            internal const ulong Basic = 1 << 0;
+            internal const ulong Digest = 1 << 1;
+            internal const ulong Negotiate = 1 << 2;
+            internal const ulong Ntlm    =  1 << 3;
```

Comment:
NTLM should not be listed.

Winhttphandler diesnt support it.  It had security issues.

Negotiate will fallback to NTLM but NTLM should be accepted as a scheme